server/utils/MailService.test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env jest */
+import MailService from './MailService';
+import config from '../../yeep.config';
+
+describe('MailService', () => {
+  let mailService;
+
+  beforeAll(async () => {
+    mailService = new MailService({
+      ...config.mail,
+    });
+  });
+
+  afterAll(async () => {
+    mailService.disconnect();
+  });
+
+  describe('sendMail', () => {
+    test('sends email', async () => {
+      await mailService.sendMail({
```

Comment:
I believe you should be testing for not throwing an error. I will try to find you a handy snippet.